private/locale-pack/index.mjs

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable no-console, prefer-arrow-callback */
+import path from 'node:path'
+import fs from 'node:fs'
+
+import dedent from 'dedent'
+import stringifyObject from 'stringify-object'
+import { remark } from 'remark'
+import { headingRange } from 'mdast-util-heading-range'
+import remarkFrontmatter from 'remark-frontmatter'
+
+import remarkConfig from '../remark-lint-uppy/index.js'
+
+import {
+  readFile,
+  writeFile,
+  getPaths,
+  sortObjectAlphabetically,
+} from './helpers.mjs'
+
+const { settings: remarkSettings } = remarkConfig
+
+const root = path.join('..', '..')
+const localesPath = path.join(root, 'packages', '@uppy', 'locales')
+const templatePath = path.join(localesPath, 'template.js')
+const englishLocalePath = path.join(localesPath, 'src', 'en_US.js')
+
+main()
+  .then(() => {
+    console.log(`✅ Generated '${englishLocalePath}'`)
+    console.log('✅ Generated locale docs')
+    console.log('✅ Generated types')
+  })
+  .catch((error) => {
+    console.error(error)
+    process.exit(1)
+  })
```

Comment:
I don't mind the `main` function and the usage of `.then` in this case. In needs to be wrapped in try catch otherwise which is makes it about the same again.

Reply:
Well you don't need a `try/catch` here...

Reply:
I need to catch it somewhere, right? Maybe I don't understand.

Reply:
```console
$ node --input-type=module -e 'await Promise.reject(new Error("test"))'
file://…/[eval1]:1
await Promise.reject(new Error("test"))
                     ^

Error: test
    at file://…/[eval1]:1:22
    at ModuleJob.run (node:internal/modules/esm/module_job:183:25)
    at async Loader.eval (node:internal/modules/esm/loader:170:24)
    at async node:internal/process/execution:49:24
    at async loadESM (node:internal/process/esm_loader:68:5)
    at async handleMainPromise (node:internal/modules/run_main:63:12)
$ node --input-type=module -e 'try { await Promise.reject(new Error("test")) } catch (e) { console.error(e); process.exit(1) }'
Error: test
    at file://…/[eval1]:1:28
    at ModuleJob.run (node:internal/modules/esm/module_job:183:25)
    at async Loader.eval (node:internal/modules/esm/loader:170:24)
    at async node:internal/process/execution:49:24
    at async loadESM (node:internal/process/esm_loader:68:5)
    at async handleMainPromise (node:internal/modules/run_main:63:12)
$ node --input-type=module -e 'Promise.reject(new Error("test")).catch(e => { console.error(e); process.exit(1) })'
Error: test
    at file:///Users/duhamean/[eval1]:1:16
    at ModuleJob.run (node:internal/modules/esm/module_job:183:25)
    at async Loader.eval (node:internal/modules/esm/loader:170:24)
    at async node:internal/process/execution:49:24
    at async loadESM (node:internal/process/esm_loader:68:5)
    at async handleMainPromise (node:internal/modules/run_main:63:12)
```

It all looks pretty similar to me, but I have a clear preference for the top-level await version without the `try/catch`.